### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by bkeepers** [devs-who-sail/discussions#1] [General]
Welcome to the Devs Who Sail community! Take a moment to introduce yourself:

1. Who are you?
2. What is your software experience?
3. What is your sailing experience?
4. What is at least thing you hope to learn from this community?

**Comment by bkeepers** [comment]
Hello! I am Brandon Keepers (@bkeepers on GitHub, [twitter](https://twitter.com/bkeepers), [Reddit](https://reddit.com/u/bkeepers), etc.).

I've been writing software professionally since 2002. I spent the last 7 years working at GitHub—in various engineering, product, and marketing roles—working on things like [GitHub Actions](https://github.com/features/actions), [Probot](https://probot.github.io), and the [GitHub + Slack integration](https://slack.github.com).

After falling in love with sailing in 2017, I bought a sailboat and spent the summer of 2018 living on, working from, and [cruising around Lake Michigan on a Catalina 36 with my wife and dog](https://www.facebook.com/svadalovelace).

I'm looking forward to learning from the community on a ton of topics, like getting a good internet connection on your boat, mobile apps you use on your boat, and the software you wish existed.

I started this community because I wanted a place to chat with people about sailing and software and it [appeared](https://www.reddit.com/r/sailing/comments/aejr9z/any_other_software_developers_who_sail_around_here/) that nothing else like this existed. I'm looking forward to learning from you!

**Comment by wolfhoundjesse** [comment]
Hello! You'll find me as Jesse M. Holmes (@wolfhoundjesse) most places.

## A Brief Detour

I started coding when I was 10, but when I was derailed when I discovered music in college. I ended up with several almost degrees and a bachelors in music performance, and I skipped out on tech to travel the world as a tuba player for the U.S. Army. I can tell this story twenty different ways, and it still won't make sense how I ended up as a software developer in Annapolis, MD.

## Back to Tech

A rare opportunity presented itself in 2015, and I became the co-owner of the information system running all of Army Music. It's a 20-year-old code base built mostly on ASP.Net, and when I came on board, I was reluctant to deal with Microsoft at all. My prior experience, which at that point was nearly ten years old, was all PHP and C++. Then I spent 2015–2018 building Angular applications on top of ASP.Net Core from a very powerful ArchLinux laptop. I _still_ miss that Dell Precision 5520. In October of 2018 I started at RxNT building ambulatory healthcare software for a company with an amazing history, a great work culture, and good people.

## What About Sailing?

I moved to D.C. Metro area in June, 2015. This place wasn't even on my radar. Just 30 days after I moved to Bowie, MD, I took an exit for Annapolis, parked on main street, got out of the car, turned around to see the harbor, and I knew I'd be moving here as soon as my lease was up 11 months later. 

When I was 4, I was thrown overboard when my parents attempted to bank a flat-bottom boat in the dark after a night bullfrogging on the Brazos River. I remember taking a breath, hooking my arms around my cowboy boots, and flipping into and under the water for an eternity. Of course, my stepdad pulled me out in less than two seconds with his incredible dad reflexes and hoisted me six feet in the air onto his shoulders. The experience made me weary of deep water for decades.

One trip on the Schooner Woodwind II changed everything. 

In 2017 I went out a few times with a coworker. In 2018 we participated in a beer can racing league. This year I'm determined to be on the water several times a month to continue learning as much as possible. 

By the way, @bkeepers, I found this by searching for sailing topics on GitHub. I work with a guy who has been building software at Quantum for 20 years, and I've been looking for some projects to connect me with other sailors/future sailors.

Nice to meet you!

**Comment by andrewbredow** [comment]
Hello! I'm @andrewbredow (pretty much everywhere on the internet 😄). I've been a software developer since about 2006 - I've worked for startups, consultancies, and product companies. Currently, I'm working as an Engineering Manager at GitHub, leading a team in the security product space.

In 2017, I bought a small 13' [Lone Star 13](https://sailboat.guide/chrysler/lone-star-13) somewhat on a whim and fell in love with sailing very quickly. During non-covid times I crew and race on an [S2 7.9](https://sailboat.guide/s2/79), which has been a huge factor in getting up-to-speed on my sailing! In 2019, I bought and refit a 1977 [Catalina 22](https://sailboat.guide/catalina/22), Athena, which my family and I sail out of Holland, MI.

I'm looking forward to seeing this community develop. First, most boating technology and software is pretty awful - I'd love to see new solutions to some of the traditional problems in this space. Second, I love the community around sailing, and I'm looking forward to connecting with other software developers who share this passion!
